### PR TITLE
Add assets deploy task. Fix styles to be more contained. Add uppercase o...

### PR DESCRIPTION
**Pull request by davidherse**
...ption to buttons

### CODE REVIEW

File: mixins/assests.js [hunk 1]
```diff
@@ -3,9 +3,9 @@
...
 
 module.exports = {
-  assestPath: function(assestName) {
+  assetsPath: function(assetsName) {
     var sever =  process.env.BUILD_ENV === 'package' ? '/' : pkg.assets_cdn;
 
-    return [sever, 'images-', pkg.version, '/', assestName].join('');
+    return [sever, 'images-', pkg.version, '/', assetsName].join('');
   },
 };
```
Comment (slwen) [inline mixins/assests.js]: Not a comment for this line but this file name appears to be misspelled.

File: package.json [hunk 2]
```diff
@@ -19,7 +19,8 @@
     "test": "node_modules/.bin/jest",
     "start": "node server.js",
     "build": "node_modules/.bin/gulp build",
-    "postinstall": "npm run build"
+    "postinstall": "npm run build",
+    "deploy_assetss": "bin/deploy_assetss"
```
Comment (slwen) [inline package.json]: Extra "s" on the end of assets?

File: mixins/assests.js [hunk 3]
```diff
@@ -3,9 +3,9 @@
 var pkg = require('../package');
 
 module.exports = {
-  assestPath: function(assestName) {
+  assetsPath: function(assetsName) {
```
Comment (bjbrewster) [inline mixins/assests.js]: LOL, copy/paste much?

Can you rename the file to assets.js too please?

File: package.json [hunk 4]
```diff
@@ -19,7 +19,8 @@
     "test": "node_modules/.bin/jest",
     "start": "node server.js",
     "build": "node_modules/.bin/gulp build",
-    "postinstall": "npm run build"
+    "postinstall": "npm run build",
```
Comment (bjbrewster) [inline package.json]: post_install?

File: package.json (hunk 2 above)
Comment (bjbrewster) [inline package.json]: Should the script name and filename be `deploy_assets`, or is there a reason for extra S in `assetss`?  Does your keyboard have a sticky `S` key?

File: gulpfile.js [hunk 5]
```diff
@@ -1,51 +1,35 @@
...
-  'typography/**/*.{type}'
-];
-
-process.env.NODE_ENV = debug ? 'development' : 'production';
+var modulePathsWithType = require('./gulp_tasks/modulePathsWithType');
 
-function modulePathsWithType(type) {
-  var paths = [];
+// Tasks
+require('./gulp_tasks/assets-deploy.js');
```
Comment (bjbrewster) [inline gulpfile.js]: I like how you've extracted some of the larger tasks out into separate files.

File: package.json (hunk 4 above)
Comment (davidherse) [inline package.json]: It's for Heroku. Runs after install to create a build.

File: package.json (hunk 2 above)
Comment (davidherse) [inline package.json]: Find replace fail

File: gulpfile.js (hunk 5 above)
Comment (davidherse) [inline gulpfile.js]: Could go further. Will keep on moving them.

File: package.json (hunk 4 above)
Comment (davidherse) [inline package.json]: This is not an addition in this PR.

File: sass/modules/_resets.scss [hunk 6]
```diff
@@ -4,6 +4,8 @@
   outline: 0;
   text-decoration: none;
   box-sizing: border-box;
+  font-family: lato,helvetica,arial,sans-serif;
```
Comment (bjbrewster) [inline sass/modules/_resets.scss]: "Lato", Helvetica, Arial, sans-serif and variants thereof are defined in many places.  Can they please use a common variable in sass/_variables.scss to make it easier to update?

File: package.json (hunk 4 above)
Comment (bjbrewster) [inline package.json]: ah, ok

**Comment by bjbrewster** [comment]
:+1:

**Pull request merged**